pubspec.yaml

```diff
@@ -0,0 +1,90 @@
+name: frontend
```

Comment:
Set the name to HolyShift.

Also adjust the name wherever needed to make the App name HolyShift in the stores and when installed.